packages/server/src/api/routes/tests/view.spec.ts

```diff
@@ -73,25 +73,12 @@ describe("/views", () => {
   }
 
   describe("create", () => {
-    it("returns a success message when the view is successfully created", async () => {
-      await saveView()
-      expect(events.view.created).toHaveBeenCalledTimes(1)
-    })
-
     it("creates a view with a calculation", async () => {
       jest.clearAllMocks()
 
       const view = await saveView({ calculation: ViewCalculation.COUNT })
 
       expect(view.tableId).toBe(table._id)
-      expect(events.view.created).toHaveBeenCalledTimes(1)
```

Comment:
How come all the deletions here? (not that I'm not pleased to see mocks getting deleted)

Reply:
These events are no longer tracked for the legacy views - we just track them in v2 now